services/ui-src/src/components/pages/Dashboard/DashboardPage.tsx

```diff
@@ -237,22 +243,34 @@ export const DashboardPage = ({ reportType }: Props) => {
     <PageTemplate type="report" sx={sx.layout}>
       <Link as={RouterLink} to="/" sx={sx.returnLink}>
         <Image src={arrowLeftIcon} alt="Arrow left" className="returnIcon" />
-        Return Home
+        Return home
       </Link>
       {errorMessage && <ErrorAlert error={errorMessage} />}
+      {/* Only show SAR alert banner if the corresponding Work Plan is not approved */}
+      {reportType === ReportType.SAR &&
+        workPlanStatus !== ReportStatus.APPROVED && (
+          <Alert
+            title={sarVerbiage.alertBanner.title}
+            showIcon={true}
+            icon={alertIcon}
+            status={AlertTypes.ERROR}
+            description={sarVerbiage.alertBanner.body}
+            sx={sx.alertBanner}
+          />
+        )}
       <Box sx={sx.leadTextBox}>
         <Heading as="h1" sx={sx.headerText}>
           {fullStateName} {intro.header}
         </Heading>
-        {reportType === "WP" && (
-          <InstructionsAccordion
-            verbiage={
-              userIsAdmin
-                ? accordion.WP.adminDashboard
-                : accordion.WP.stateUserDashboard
-            }
-          />
-        )}
+        <InstructionsAccordion
+          verbiage={
+            userIsAdmin
```

Comment:
I think we need different instructions for admin WP and admin SAR

Reply:
heya! so the `reportType` index will pull in the correct instructions from the verbiage file, it should work as expected (once I put in the verbiage for admin `SAR`)